test/RoboCop.ts

```diff
@@ -297,10 +297,10 @@ describe("RoboCop", () => {
       const { ruleMakerWallet, roboCop } = await deployValidRuleFixture();
       // these error because the non existent hash doesnt belong to the subscriber
```

Comment:
this comment isnt applicable anymore. We can just remove it since the current error message is generic and makes sense